docroot/themes/custom/uids_base/scss/components/menus/off-canvas.scss

```diff
@@ -1,6 +1,23 @@
-@import "assets/scss/_variables.scss";
-@import "assets/scss/_utilities.scss";
-@import "components/menus/off-canvas/off-canvas.scss";
+@import "uids3/assets/scss/_variables.scss";
+@import "uids3/assets/scss/_utilities.scss";
+@import "uids3/components/menus/off-canvas/off-canvas.scss";
+
+.bttn--drawer {
+  border: none;
+  display: inline-block;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+  cursor: pointer;
+  text-align: center;
+  text-decoration: none;
+  text-transform: uppercase;
+  line-height: 1.4;
+  padding: 1rem 2rem;
+  -webkit-font-smoothing: auto;
+  -moz-osx-font-smoothing: auto;
+}
```

Comment:
Is this something we need a special class for? Or can we modify it by its presence in the off canvas area? Something like `.off-canvas .bttn`?